### DISCUSSION TITLE
WikiJS is no longer functional on AWS

### DISCUSSION THREAD

**Discussion by ewirtz65** [requarks/wiki#5186] [Help / Questions]
I have an installation from the Marketplace that has been running for over a year with no issues.  I cannot access it anymore.  But I do not know where to start trouble shooting at this point.  I would not mind doing a manual re-install but I am afraid of loosing all the content, much of which only exists within the wiki..  Seems like I tried to do an update a month or so ago and it never came back up.
I know very little about docker and finding information about where things were install through the marketplace are non-existent.
I suspect it is in an unstable state from a failed update.  I do see that others are having issues on AWS as well, but no answers have been posted for them.

One potential issue I see from running a ps command is that the container IP in this command it NOT the server IP:
 /usr/bin/docker-proxy -proto tcp -host-ip 0.0.0.0 -host-port 80 -container-ip 172.18.0.4 -container-port 3000
But I do not know where the file is that would need to be updated.

I need help getting it back up and running.
Docker version 19.03.12, build 48a66213fe
Ubuntu 18.04.4 LTS
Wiki version?? I don't know where to find this.

**Comment by NGPixel** [comment]
The AWS image was using the exact same instructions as detailed in https://docs.requarks.io/install/ubuntu

You can recreate the container without loosing any data by running:
```sh
docker stop wiki
docker rm wiki
docker pull ghcr.io/requarks/wiki:2
docker run -d --name=wiki -e DB_TYPE=postgres -e DB_HOST=db -e DB_PORT=5432 -e DB_PASS_FILE=/etc/wiki/.db-secret -v /etc/wiki/.db-secret:/etc/wiki/.db-secret:ro -e DB_USER=wiki -e DB_NAME=wiki -e UPGRADE_COMPANION=1 --restart=unless-stopped -h wiki --network=wikinet -p 80:3000 -p 443:3443 ghcr.io/requarks/wiki:2
```

Give it a minute or two to initialize. You can see if started correctly using `docker logs wiki`
